### PR TITLE
Fix dependency to sensio/framework-extra-bundle

### PR DESCRIPTION
**Pull request by rokkie**
\FOS\RestBundle\Request\ParameterBag has a dependency to
\Doctrine\Common\Util\ClassUtils which originates from doctrine/common
which in turn a dependency of sensio/framework-extra-bundle but this is
declared as a dev-dependency. This causes an fatal error with the
message 'Class 'Doctrine\Common\Util\ClassUtils' not found'.

Fixes#259

### CODE REVIEW

**Comment by xabbuh** [comment]
I would rather add `doctrine/common` as a dependency then because that is what we need here.

**Comment by rokkie** [comment]
Changed the dependency to `doctrine/common`.

**Comment by xabbuh** [comment]
Thinking about this again, I wonder if we just modify the code instead. I mean we do not need to use the `ClassUtils::getClass()` method if Doctrine is not installed. In these cases, just using `get_class()` should be sufficient.

File: Request/ParameterBag.php [hunk 1]
```diff
@@ -11,7 +11,6 @@
 
 namespace FOS\RestBundle\Request;
 
-use Doctrine\Common\Util\ClassUtils;
```
Comment (xabbuh) [inline Request/ParameterBag.php]: this should be reverted

File: Request/ParameterBag.php [hunk 2]
```diff
@@ -76,8 +77,12 @@ private function initParams($requestId)
             );
         }
 
+        $cls = class_exists('\Doctrine\Common\Util\ClassUtils', true)
```
Comment (xabbuh) [inline Request/ParameterBag.php]: you can use `class_exists(ClassUtils::class)` here then

**Comment by GuilhemN** [comment]
Thank you @rokkie!

**Pull request merged**